endpoints/_includes/functions.php

```diff
@@ -736,20 +736,18 @@ function helplineSearch($latitude, $longitude)
 
 function isBMLTServerOwned($latitude, $longitude)
 {
-    $bmlt_search_endpoint = sprintf('%s/client_interface/json/?switcher=GetSearchResults&data_field_key=root_server_uri&sort_results_by_distance=1&long_val=%s&lat_val=%s&geo_width=%s',
+    $bmlt_search_endpoint = sprintf('%s/client_interface/json/?switcher=GetSearchResults&data_field_key=root_server_uri&sort_results_by_distance=1&lat_val=%s&long_val=%s&geo_width=%s',
         setting('tomato_url'), $latitude, $longitude, setting('helpline_search_radius'));
     $search_results = json_decode(get($bmlt_search_endpoint));
     $root_server_uri_from_first_result = $search_results[0]->root_server_uri;
-    return strpos(getAdminBMLTRootServer(), $root_server_uri_from_first_result) == 0;
+    return ($root_server_uri_from_first_result == getAdminBMLTRootServer().'/');
 }
 
 function getHelplineRoutingBMLTServer($latitude, $longitude) {
     if (setting('tomato_helpline_routing') & !isBMLTServerOwned($latitude, $longitude)) {
         return setting('tomato_url');
-    } else if (has_setting('helpline_bmlt_root_server')) {
-        return setting('helpline_bmlt_root_server');
     } else {
-        return setting('bmlt_root_server');
+        return getAdminBMLTRootServer();
```

Comment:
Not seeing the value of this change.

Reply:
Just refers to the helpline setting check - if that’s not needed, not a big deal

Reply:
This change makes sense.